components/image/image.directive.ts

```diff
@@ -168,4 +189,11 @@ export class NzImageDirective implements OnInit, OnChanges, OnDestroy {
         });
     }
   }
+
+  private backLoadCompleteStateHandler(): void {
+    this.status = 'normal';
+    this.getElement().nativeElement.loading = this.nzLoading;
```

Comment:
This attribute `loading` is not supported by chrome browsers lower than version 77, so it's better to not add it if `nzLoadingType` is not set. (https://caniuse.com/?search=eager)

Reply:
I apologize for the delay.
This could be removed entirely, as setting the image element's lazy loading does not produce the desired outcome.